runtime/include/cloe/data_broker.hpp

```diff
@@ -576,7 +577,7 @@ class Signal {
     * Adds a name of the signal
     * \param name Name of the signal
     */
-  void add_name(std::string name) { names_.emplace_back(std::move(name)); }
+  void add_name(std::string_view name) { names_.emplace_back(std::move(name)); }

```

Comment:
Using string_view, the move is obsolete :)

Reply:
Done.